README.md

```diff
@@ -8,16 +8,19 @@ The IBM i SQLAlchemy adapter provides a [SQLAlchemy](https://www.sqlalchemy.org/
 **Please note that this project is still in active development and is not ready for use.** :rotating_light: 
 
 ```python
-from sqlalchemy import create_engine, db
+import sqlalchemy as sa
 
+engine = sa.create_engine("ibmi://user:pass@host[:port]/database")
```

Comment:
```suggestion
engine = sa.create_engine("ibmi://user:pass@host/database")
```

There's no way to specify the port for the ODBC driver.

Reply:
We should probably omit the database as well. The driver will connect to the default *LOCAL database by default, which is what 99% of users will want. I _think_ you still need the slash, though.